livekit-agents/livekit/agents/pipeline/pipeline_agent.py

```diff
@@ -1109,12 +1109,11 @@ class _DeferredReplyValidation:
     PUNCTUATION = ".!?"
     PUNCTUATION_REDUCE_FACTOR = 0.75
 
-    LATE_TRANSCRIPT_TOLERANCE = 1.5  # late compared to end of speech
-
     # Long delay to use when the model thinks the user is still speaking
-    # TODO: make this configurable
     UNLIKELY_ENDPOINT_DELAY = 6
 
+    FINAL_TRANSCRIPT_TIMEOUT = 5
```

Comment:
5 seems like a lot, what about 2?

Reply:
STT could be slow sometimes, and not sending a final one is rare.. so trying to make this a last resort kind of thing